cr/cr-core/scripts/seed_lk_100k.py

```
@@ -0,0 +1,285 @@
+    }
+
+
+def mutate_for_duplicate(patient: dict) -> dict:
+    """Return a realistic duplicate: always a NEW MRN (separate registration event),
+    NIC retained only 60 % of the time, plus one demographic variation."""
+    import copy
+    dup = copy.deepcopy(patient)
+
+    # Every duplicate originates from a separate registration — always a new MRN,
+    # possibly at a different hospital.
+    hosp        = random.choice(HOSPITALS)
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)